### DISCUSSION TITLE
How to add more feature to saleor ?

### DISCUSSION THREAD

**Discussion by phearakthefoundry** [saleor/saleor#6468] [Q&A]
Currently I would like to add one more feature on saleor-dashboard.
The feature is add (banner, company_name).
For the flow of feature : 
when admin dashboard add new banner for company the banner should be show on store front by specific company name.

Example: Admin dashboard add new banner( Cat)  for company A , and banner ( Fish ) for company B .So on store front of company A show banner ( Cat ) and store-front of company B show banner( Fish ).

I'm a new developer so need all of you help to tell me step by step.

Thank you

**Comment by karolkielecki** [comment]
Hey! I think you can do that on the storefront level. Check the following spectrum thread about how to modify the storefront: https://spectrum.chat/saleor/saleor-storefront/modifying-the-storefront~c1955dbf-a421-4fb6-b99e-937dd2642b23

**Reply by phearakthefoundry** [reply]
admin would like to manage banner like (create, update , delete) from saleor-dashboard  and saleor-storefront just get banner to display from admin.
currently I have two type of users such as admin, user .
admin manage saleor-dashboard and user manage saleor-storefront.
